scripts/pre-build.js

```diff
@@ -84,6 +82,39 @@ const start = async (source) => {
 
   console.log('Updating sidebar.js');
   await createSidebar('docs', path.join(process.cwd(), 'sidebars.js'));
+
+  console.log('Downloading translations');
+  const locales = await downloadTranslations(I18N_FOLDER);
+
+  for (const locale of locales) {
+    const localeDocs = path.join(
+      I18N_FOLDER,
+      locale,
+      'docusaurus-plugin-content-docs',
+      'current'
+    );
+    const staticResources = ['fiddles', 'images'];
```

Comment:
Yes and no. Docusaurus is built in such a way that each locale is independent to enable exactly that.
The problem right now is that our i18n pipeline doesn't upload any of those assets to Crowding, so when I download the translations they are missing and I have to "manually" copy them from the source or otherwise they will be missing.